frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -165,14 +164,11 @@ class PopupCreateFile extends React.Component {
 
     const uploadInProgress = e => {
       if (e.lengthComputable) {
-        const uploadFiles = state.uploadFiles
-        uploadFiles[uploadFiles.indexOf(file)].percent = (e.loaded / e.total * 100) / uploadFiles.length
+        const uploadFilesInProgress = state.uploadFiles
+        uploadFilesInProgress[uploadFilesInProgress.indexOf(file)].percent = (e.loaded / e.total * 99) / uploadFilesInProgress.length
```

Comment:
refactor this and factorize it with other functions that calculate the percent of the upload